### PR TITLE
Setup postgresql

### PR DESCRIPTION
**Pull request by andrevieira11**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,8 +1,37 @@
...
+services:
+  revent-db:
+    container_name: revent-db
+    image: postgres:latest
+    # restart: always
+    env_file:
+      - ".env"
+    ports:
+      - "5500:5432"
+    volumes:
```
Comment (kenvontucky) [inline docker-compose.yml:11]: volume is not necessary

File: docker-compose.yml [hunk 2]
```diff
@@ -1,8 +1,37 @@
-# version: '3.9'
+version: "3.9"
+services:
+  revent-db:
+    container_name: revent-db
+    image: postgres:latest
+    # restart: always
```
Comment (kenvontucky) [inline docker-compose.yml:6]: comment can be removed

File: docker-compose.yml [hunk 3]
```diff
@@ -1,8 +1,37 @@
...
+    volumes:
+      - ./data/db:/var/lib/postgresql/data
 
-# services:
-#   web:
-#     build: .
-#     command: python manage.py runserver 0.0.0.0:8000
-#     ports:
-#       - 8000:8000
+  revent-be:
```
Comment (kenvontucky) [inline docker-compose.yml:14]: what is the difference between this service and web? do we need this?

**Review by kenvontucky — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 3 above)
Comment (andrevieira11) [reply docker-compose.yml:11]: removed web instead, is it ok?

File: docker-compose.yml [hunk 4]
```diff
@@ -1,8 +1,12 @@
...
+    ports:
+      - "5432:5432"
 
-# services:
-#   web:
-#     build: .
-#     command: python manage.py runserver 0.0.0.0:8000
-#     ports:
-#       - 8000:8000
+volumes:
```
Comment (kenvontucky) [inline docker-compose.yml:11]: this part is not necessary

File: revent/settings.py [hunk 5]
```diff
@@ -41,18 +41,25 @@
...
     "django.middleware.common.CommonMiddleware",
     "django.middleware.csrf.CsrfViewMiddleware",
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
 ]
 
+CORS_ALLOWED_ORIGINS = [
+    "http://localhost:3000",
+    "http://localhost:3001",
```
Comment (kenvontucky) [inline revent/settings.py:60]: why do we need 3000 and 3001, isn't just one enough?

**Review by kenvontucky — APPROVED** [review]

**Pull request merged**